lib/elixir/test/elixir/tuple_test.exs

```diff
@@ -62,4 +62,11 @@ defmodule TupleTest do
     mod = Tuple
     assert mod.delete_at({:foo, :bar, :baz}, 0) == {:bar, :baz}
   end
+
+  test "product/1" do
+    assert Tuple.product({255, 255}) == 65025
+
+    mod = Tuple
+    assert mod.product({255, 255}) == 65025
```

Comment:
This test is not needed.

Reply:
I should clarify that the test itself is needed, only the assertion with dynamic module is not needed.

Reply:
@ericmj Yes, that's correct - I removed whole test, because there are also `doctests`, but if you say so I will put test back.